source/_docs/unsupported-modules-plugins.md

```diff
@@ -445,6 +445,19 @@ The list of [WordPress roles and capabilities](https://codex.wordpress.org/Roles
 * `edit_plugins`
 * `edit_themes`
 
+## Paid Plugins or Licenses
+
+In general, licensed or paid plugins, modules or themes are allowed for use in Pantheon. For their use, consider platform constrants or conflicts that will arise due to the nature of the platform. Licensing issues are for Customers to Resolve, as per the terms of their plugin or themes.
+
+Paid tools or licenses usually get direct support from developers. It is also used to get updates directly online. Most of the time, you update your code in DEV, and just commit the changes to TEST and LIVE.
```

Comment:
You want to update code to DEV *all of the time*. This isn't specific to plugins with licenses.